app.js

```diff
@@ -27,16 +28,23 @@ app.get("/api/articles/:article_id/comments", getCommentsByArticleId);
 
 app.post("/api/articles/:article_id/comments", postCommentToArticleById);
 
+app.delete("/api/comments/:comment_id", deleteCommentById);
+
 app.use((req, res, next) => {
   res.status(404).send({ msg: "Route not found" });
   next(err);
 });
 
 app.use((err, req, res, next) => {
   if (err.code === "22P02") {
-    res.status(400).send({ msg: "Article ID is invalid" });
-  }
-  next(err);
+    if (req.path.includes("/articles/") && req.path.includes("/comments/")) {
+      res.status(400).send({ msg: "Article ID is invalid" });
+    } else if (req.path.includes("/articles/")) {
+      res.status(400).send({ msg: "Article ID is invalid" });
+    } else if (req.path.includes("/comments/")) {
+      res.status(400).send({ msg: "Comment ID is invalid" });
+    }
```

Comment:
We can set up our errors handlers to look for the presence of a msg property and then we can return the status and msg that we Promise Rejected.